Jenkinsfile

```diff
@@ -222,6 +223,46 @@ pipeline {
             }
           }
         }
+        stage('ARM build-test') {
+          agent { label 'arm' }
+          options {
+            skipDefaultCheckout()
+            warnError('ARM execution failed')
+          }
+          when {
+            beforeAgent true
+            anyOf {
+              expression { return env.GITHUB_COMMENT?.contains('arm tests') || env.GITHUB_COMMENT?.contains('/arm')}
+              branch 'master'
```

Comment:
@axw , do we want to enable the `arm` builds in 7.x?

Reply:
This is just running tests right? I think it's enough to only run them on master.

Reply:
Exactly, it builds and runs the unit tests